EnrollmentFlows/SignUp/header.html

```diff
@@ -0,0 +1 @@
+<div class="corner-ribbon red">Demo</div>
```

Comment:
I would turn this into a link:
`header.html --> ../header.html`
unless the Enrollment UI theme header is different

Reply:
Done